components/api/src/main/java/com/hotels/styx/api/NettyCookie.java

```diff
@@ -69,7 +69,7 @@ public String toString() {
             buf.append(", HTTPOnly");
         }
         if (sameSite != null) {
-            buf.append(", SameSite=").append(sameSite.toString());
+            buf.append(", SameSite=").append(sameSite);
```

Comment:
This is out of place.

Reply:
I missed pushing that small change, so I just let it go through this time.  I can rollback if you prefer it.